### PR TITLE
add variables colors, text, font-size

### PR DESCRIPTION
**Pull request by ceridwenroberts**

### CODE REVIEW

File: src/app/globals.css [hunk 1]
```diff
@@ -1,8 +1,42 @@
...
+/* font-size */
+
+  --font-size-base: 16px;
+  --font-size-h1: 32px;
+  --font-size-h2: 24px; 
+  --font-size-footer: 14px;
+
+/* colours */
+
+  --blue-dark: #2C3386;
```
Comment (milutxi) [inline src/app/globals.css]: need to add c- like in the other colors

File: src/app/globals.css [hunk 2]
```diff
@@ -1,8 +1,42 @@
...
+  --btn-orange: var(--c-orange);
+
+  --c-text-dark: #000;
+  --c-text-light: #fff;
+
+/* fonts */
+
+  --font-h1: 'Poppins', sans serif;
+  --font-h2: 'Kumbh Sans', sans serif;
+  --font-btn: 'Kumbh Sans', sans serif;
```
Comment (milutxi) [inline src/app/globals.css:41]: these fonts are downloaded?

File: src/app/globals.css [hunk 3]
```diff
@@ -1,8 +1,42 @@
...
+  --blue-dark: #2C3386;
+  --c-blue-light: #97B8D7;
+  --c-blue: #3449FF;
+  --c-grey-dark: #CCCCCC;
+  --c-green: #7ACD6D;
+  --c-grey-light: #EDEDED;
+  --c-orange: #F4770A;
+  --c-red: #EA4554;
+  --c-violet: #9AA4FF;
+
```
Comment (milutxi) [inline src/app/globals.css:28]: it need one color more, #B7B7B7, (is the border color)

**Comment by ceridwenroberts** [comment]
Please recheck my code <3

**Review by Sakas19 — APPROVED** [review]

**Pull request merged**